windows/devices/uwb/UwbConnector.cxx

```diff
@@ -524,12 +524,51 @@ UwbConnector::SessionGetRangingCount(uint32_t sessionId)
     return resultFuture;
 }
 
-std::future<UwbSessionUpdateMulicastListStatus>
+std::future<UwbStatus>
 UwbConnector::SessionUpdateControllerMulticastList(uint32_t sessionId, UwbMulticastAction multicastAction, std::vector<::uwb::UwbMacAddress> controlees)
 {
-    std::promise<UwbSessionUpdateMulicastListStatus> resultPromise;
+    std::promise<UwbStatus> resultPromise;
     auto resultFuture = resultPromise.get_future();
-    // TODO: invoke IOCTL_UWB_SESSION_UPDATE_CONTROLLER_MULTICAST_LIST
+
+    wil::unique_hfile handleDriver;
+    auto hr = OpenDriverHandle(handleDriver, m_deviceName.c_str());
+    if (FAILED(hr)) {
+        PLOG_ERROR << "failed to obtain driver handle for " << m_deviceName << ", hr=" << std::showbase << std::hex << hr;
+        resultPromise.set_exception(std::make_exception_ptr(UwbException(UwbStatusGeneric::Rejected)));
+        return resultFuture;
+    }
+
+    UwbSessionUpdateMulicastList multicastList{
+        .SessionId = sessionId,
+        .Action = multicastAction,
+    };
+
+    for (const auto& controlee : controlees) {
+        multicastList.Controlees.push_back(UwbSessionUpdateMulticastListEntry{
+            .ControleeMacAddress = controlee,
+        });
+    }

```

Comment:
nit: this can be done with a transform:

```suggestion
    std::ranges::transform(controlees, std::back_inserter(multicastList.Controlees), [](const auto &controlee) {
        return UwbSessionUpdateMulticastListEntry{
            .ControleeMacAddress = controlee,
        });
```